package.json

```diff
@@ -13,6 +13,7 @@
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@wordpress/babel-preset-default": "^1.1.2",
     "@wordpress/block-serialization-spec-parser": "^1.0.0",
+    "@wordpress/scripts": "^2.4.0",
```

Comment:
Not sure what is wrong with including `@wordpress/scripts` as a dep (`yarn install` fails with `error Package "" refers to a non-existing file '"<path to project root>/babel-preset-default"'`) but, do we really need to add this dep? Building and running the app seems fine even without it.